### ISSUE TITLE
Typography/Body Copy - Line Length

### ISSUE THREAD

**Issue by jinky32** [ThreeUK/digital-library.three.co.uk#10]
**Requirement**: the width of a block of text (with default settings in common browsers) should not exceed 80 characters.

**Why is this required?** People with visual impairment and dyslexia find it difficult to keep their place in blocks of text displayed in long lines.

**Comment by papafed** [comment]
This is a nonsense requirement that we can never adhere to, because how wide is 80 characters? How wide is an M and how wide is a I? Clearly written by someone with absolutely no of typography or CSS.
There is no way in CSS to restrict width by characters anyway, only metrics (pixels, percents, etc), so it cannot be achieved.
